packages/dbos-cloud/cli.ts

```diff
@@ -249,6 +249,17 @@ databaseCommands
     process.exit(exitCode);
   }))
 
+databaseCommands
+  .command('restore')
+  .description("Restore a Postgres database instance to a specified time")
+  .argument('<name>', 'database instance name')
+  .requiredOption('-t, --restore-time <string>', 'Specify the timestamp to restore from, must be in RFC3339 format. Example: 2009-09-07T23:45:00Z')
```

Comment:
The point in time to which to restore the database. Must be a timestamp in RFC 3339 format. Example: 2009-09-07T23:45:00Z